demo/python/basic/config.ini

```diff
@@ -0,0 +1,10 @@
+[default]
+#Current user's username
+user_name = user1
```

Comment:
Can we add newlines between each variable? i.e. comment, parameter, newline; comment, parameter, newline; etc